src/map.rs

```diff
@@ -47,25 +47,25 @@ pub struct Map {
 }
 
 impl Map {
-    pub fn center(&self) -> Translation {
+    pub fn center(&self, origin : &Translation) -> Translation {
```

Comment:
My only concern is that this likely will change quite a bit when we have parent/child relationships between the map component and the chunk components, but for now I think this works pretty well.